eva-accession-release/src/main/java/uk/ac/ebi/eva/accession/release/io/VariantContextWriter.java

```diff
@@ -56,10 +64,32 @@ public void open(ExecutionContext executionContext) throws ItemStreamException {
 
         Set<VCFHeaderLine> metaData = new HashSet<>();
         metaData.add(new VCFHeaderLine("reference", referenceAssembly));
-        metaData.add(new VCFInfoHeaderLine("VC", 1, VCFHeaderLineType.String,
+        metaData.add(new VCFInfoHeaderLine(VARIANT_CLASS_KEY, 1, VCFHeaderLineType.String,
                                            "Variant class according to the Sequence Ontology"));
-        metaData.add(new VCFInfoHeaderLine("SID", VCFHeaderLineCount.UNBOUNDED, VCFHeaderLineType.String,
+        metaData.add(new VCFInfoHeaderLine(STUDY_ID_KEY, VCFHeaderLineCount.UNBOUNDED, VCFHeaderLineType.String,
                                            "Identifiers of studies that report a variant"));
+
+        metaData.add(new VCFInfoHeaderLine(VALIDATED_KEY, 0, VCFHeaderLineType.Flag,
+                                           "If an RS was validated by any method according to dbSNP validation "
+                                           + "status, this flag will be present. If it was not validated by any "
+                                           + "method, this flag won't be present."));
+        metaData.add(new VCFInfoHeaderLine(ALLELES_MATCH_KEY, 0, VCFHeaderLineType.Flag,
+                                           "Alleles mismatch flag, present when some of the submitted variants has "
+                                           + "inconsistent allele information. See https://github"
+                                           + ".com/EBIvariation/eva-accession/wiki/Import-accessions-from-dbSNP"
+                                           + "#alleles-match"));
+
+        metaData.add(new VCFInfoHeaderLine(SUBMITTED_VARIANT_VALIDATED_KEY, VCFHeaderLineCount.UNBOUNDED,
+                                           VCFHeaderLineType.Integer,
+                                           "Number of submitted variants clustered in an RS that were validated by any"
+                                           + " method according to dbSNP validation status, the value of this field "
+                                           + "is 1, and 0 otherwise"));
+        metaData.add(new VCFInfoHeaderLine(ASSEMBLY_MATCH_KEY, 0, VCFHeaderLineType.Flag,
+                                           "Assembly mismatch flag, present when the reference allele doesn't match "
+                                           + "the reference sequence"));
+        metaData.add(new VCFInfoHeaderLine(SUPPORTED_BY_EVIDENCE_KEY, 0, VCFHeaderLineType.Flag,
+                                           "Lack of evidence. If this flag is present it means that no submitted "
```

Comment:
Shall we match the phrasing style of other flags? Something like "Lack of evidence flag, present when no submitted..."